deploy-0chain/action.yml

```diff
@@ -353,7 +353,7 @@ runs:
         count=0
         while [[ $count -le 5 ]]
         do
-          retry ./zwallet getblobbers --silent --configDir . --config ./zbox_config.yaml --wallet ../ignore | grep -o "[a-z0-9]\{64\}" > build.log
+          retry ./zbox ls-blobbers --all --silent --configDir . --config ./zbox_config.yaml --wallet ../ignore | grep -o "[a-z0-9]\{64\}" > build.log
```

Comment:
one would have assumed the impl behind the scenes would have been identical, obviously not?

Reply:
No, but this PR will make them similar https://github.com/0chain/gosdk/pull/495